docs/conf.py

```diff
@@ -0,0 +1,116 @@
+# Configuration file for the Sphinx documentation builder.
+#
+
+# -- Project information -----------------------------------------------------
+
+project = "aiida-singledocs"
+copyright = "2020, Francisco Ramirez"
+author = "Francisco Ramirez, Chris Sewell"
+
+
+version = "0.0.1"
+release = version
+
+# -- General configuration ---------------------------------------------------
+
+needs_sphinx = "2.0"
+extensions = ["myst_nb", "sphinx_panels", "ablog", "sphinx.ext.intersphinx"]
+
+myst_admonition_enable = True
+myst_html_img_enable = True
+jupyter_execute_notebooks = "auto"
+
+blog_path = "stories/index"
+blog_title = "Stories"
+blog_post_pattern = ["stories/*.md", "stories/*.ipynb"]
+post_redirect_refresh = 1
+post_auto_excerpt = 2
+post_auto_image = 1
+fontawesome_included = True
+html_sidebars = {
+    "stories/index": ["tagcloud.html", "archives.html", "sbt-sidebar-nav.html"],
+    "stories/*": [
+        "sidebar-search-bs.html",
+        "postcard.html",
+        "recentposts.html",
+        "tagcloud.html",
+        "categories.html",
+        "archives.html",
+        "sbt-sidebar-nav.html",
+        "sbt-sidebar-footer.html",
+    ],
+}
+
+intersphinx_mapping = {
+    "aiida": ("http://aiida-core.readthedocs.io/en/latest/", None),
+}
+
+# The master toctree document.
+master_doc = "index"
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path .
+exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
+
+# The name of the Pygments (syntax highlighting) style to use.
+pygments_style = "sphinx"
+
+# -- Options for HTML output -------------------------------------------------
+
+html_theme = "sphinx_book_theme"
+# html_title = f"version: {version}"
+# html_favicon = "_static/quantum-mobile-v4-text-square.png"
+# html_logo = "_static/quantum_mobile_text_wide.png"
+html_theme_options = {
+    "home_page_in_toc": True,
+    "repository_url": "https://github.com/aiidateam/aiida-blog",
+    "repository_branch": "chris-branch",
+    "use_repository_button": True,
+    "use_issues_button": True,
+    "path_to_docs": "docs",
+    "use_edit_page_button": True,
+    "launch_buttons": {
+        "binderhub_url": "https://https://mybinder.org"
+    },
+}
+panels_add_bootstrap_css = False
+
+
+def start_aiida(*args):
+    from unittest import mock
+    import os
+    import subprocess
+
+    subprocess.check_call(["reentry", "scan"])
+    from aiida.manage.external import postgres
+    from aiida.manage.tests import _GLOBAL_TEST_MANAGER, BACKEND_DJANGO
+    from aiida.common.utils import Capturing
+    # this is required on READTHEDOCS, since the docker container only contains C.UTF-8
```

Comment:
@sphuber, quick question, is there a specific reason that `_CREATE_DB_COMMAND` is hard-coded to `en_US.UTF-8`. As commented, It's the only sticking point I had on getting this to work on RTD.

I'm planning to turn this into a small sphinx-extension,
so would probably want to make a PR to aiida-core, to change this in some way, to avoid this patch

Reply:
I think we applied this to have some consistency for the databases that are created by AiiDA, but I was not very close involved in that decision. This was mostly done between @giovannipizzi and @szoupanos if I remember correctly. If I understand correctly, you would apply the same change in `aiida-core`? Instead of hard coding utf-8 as encoding, use whatever is set by `os.enivron['LANG']`. I am not sure what the consequences of this would be.

What is the exact reason why having utf-8 fails for your use case?